x-pack/plugins/ml/public/application/memory_usage/memory_tree_map/memory_page.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiCallOut, EuiSpacer } from '@elastic/eui';
+import React, { FC } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { JobMemoryTreeMap } from './tree_map';
+
+export const MemoryPage: FC = () => {
+  return (
+    <>
+      <EuiSpacer size="s" />
+      <EuiCallOut color="primary">
+        <FormattedMessage
+          id="xpack.ml.memoryUsage.treeMap.infoCallout"
+          defaultMessage="Memory usage for open jobs and trained models."
```

Comment:
Nit. The anomaly detection and data frame analytics terminology are not the same, we have open jobs for AD and started jobs for DFA. If the UI says "open", one might think it refers only to the AD jobs. Active might be a better solution as it can refer to both job types. Alternatively, we can say: "Memory usage for open or started jobs and trained models". This is the most accurate, but it's longer and harder to read.

```suggestion
          defaultMessage="Memory usage for active machine learning jobs and trained models."
```

Reply:
I'm happy with "active"